service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EhrExtractMapper.java

```diff
@@ -68,6 +74,14 @@ public EhrExtractTemplateParameters mapBundleToEhrFhirExtractParams(
         return ehrExtractTemplateParameters;
     }
 
+    private String getPatientNhsNumber(GetGpcStructuredTaskDefinition getGpcStructuredTaskDefinition) {
+        if (StringUtils.isNotBlank(overrideNhsNumber)) {
+            return overrideNhsNumber;
```

Comment:
Might be a good idea to also log a WARN level message here indicating that GP2GP_GPC_OVERRIDE_NHS_NUMBER is used instead of the real NHS numbers. Don't log the actual NHS numbers.